timeline-chart/src/layer/time-graph-chart.ts

```diff
@@ -617,10 +628,39 @@ export class TimeGraphChart extends TimeGraphChartLayer {
         el.displayObject.on('click', ((e: PIXI.InteractionEvent) => {
             if (el instanceof TimeGraphStateComponent && !this.mousePanning && !this.mouseZooming) {
                 this.selectState(el.model);
+                // Time range selection using left-click should not trigger a double click action
```

Comment:
This if-clause is for states only because it can call selectState().

But I believe that the doubleClick handling should apply for all kinds of elements (states, arrows, annotations, etc.)

Reply:
I am surprised that we get a PIXI click event when you mouseDown, mouseMove, then mouseUp (although, the mouseDown and mouseUp need to be on the same element for the click to be called).

I would propose that our click/doubleClick mouseInterationEvent be triggered only when the mouse does not move between mouseDown and mouseUp. So store the coordinates of mouseDown and it must be the same in the click event otherwise we do not propagate it. This will prevent click on range selection, in a simpler way. I would also avoid storing things related to clicks in the element itself.